lib/killbill_client/models/invoice_payment.rb

```diff
@@ -69,6 +71,22 @@ def bulk_create(external_payment = false, user = nil, reason = nil, comment = ni
                             :comment => comment,
                         }.merge(options)
       end
+
+      def chargeback_reversal(payment_id, amount, adjustments = nil, user = nil, reason = nil, comment = nil, options = {})
```

Comment:
For symmetry with the `refund` API, I would declare this as a class method above (`class << self` block) instead.